package.json

```diff
@@ -17,6 +17,8 @@
     "next": "14.2.3",
     "react": "18.3.1",
     "react-dom": "18.3.1",
+    "react-native-svg": "^15.3.0",
```

Comment:
This package is for React Native. Since we're currently developing in React for Web we do not need. Can we remove this dependency?

Reply:
Yes, I agree. Done.